XFaction/Core/Frames/ChatFrame.lua

```diff
@@ -32,6 +32,15 @@ function ChatFrame:Initialize()
                 self._ElvUIModule = ElvUI[1]:GetModule('Chat')
                 self._ChatFrameHandler = function(...) self._ElvUIModule:FloatingChatFrame_OnEvent(...) end
             end
+        elseif IsAddOnLoaded('WIM') then
```

Comment:
I personally dont use ElvUI so I assumed it was a total replacement UI and so I inserted as the second check in the if/else that existed, though I see your point that if both were installed it would be the assumption they wanted WIM as the chat handler.